File: Calculator.cpp
#include<iostream>
using namespace std;
class Calculator
{
    int num1,num2,sum,sub;
    public:void add()
    {
        cout<<"Enter 2 Numbers \n";
        cin>>num1>>num2;
        sum=num1+num2;
        cout<<"Sum is \n"<<sum<<endl;
    }
    public:void minus()
    {
        cout<<"Enter 2 Numbers \n";
        cin>>num1>>num2;
        sub=num1-num2;
        cout<<"Subtraction is \n"<<sub;
    }
};
int main()
{
    Calculator c1;
    c1.add();
    c1.minus();
}

File: JAVA/All Codes/Calculator.java
import java.util.Scanner;
class Calculator
{
public static void main(String args[])
{
int num1,num2,result,choice;
System.out.println("Enter your choice 1.Add 2.Sub....");
Scanner sc=new Scanner(System.in);
choice=sc.nextInt();
System.out.println("Enter Two Numbers:");
num1=sc.nextInt();
num2=sc.nextInt();
switch(choice)
{
case 1:result=num1+num2;
System.out.println("Result:"+result);
break;
case 2:result=num1-num2;
System.out.println("Result:"+result);
break;
case 3:default:
System.out.println("Invalid choice");
}
}
}
